Address alpine issues and credentials security (#25)
carpentries-incubator/docker-introduction: _episodes/03-running-containers.md

~~~
{: .language-bash}

> If you never used the *alpine* docker image on your computer, docker told you that it couldn't find the image and had to download it fresh.

Reviewer: ```suggestion
If you never used the *alpine* docker image on your computer, docker printed a message that it couldn't find the image and had to download it.
```

> If you did the download before, the command will probably show no output. That's because this particular container is designed for you to

Reviewer: ```suggestion
If you had used the alpine image before, the command will probably show no output. That's because this particular container is designed for you to
```

Author: simply `used` instead of `had used` ?

Reviewer: Good catch!
